db/migrate/20160720164418_add_images_to_profile.rb

```diff
@@ -0,0 +1,5 @@
+class AddImagesToProfile < ActiveRecord::Migration
+  def change
+    add_column :profiles, :images, :string, array: true, default: []
```

Comment:
Why did you take this approach, as opposed to creating a separate model for an Image? Is this in 1st normal form?